### PR TITLE
chore(project): remove unnecessary else return statement

### PR DESCRIPTION
**Pull request by josonho**
Whether to return in else increases cyclomatic complexity,

and make the level of nesting increase

***

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
LGMT, let's see if anyone in the team has comments or not.

File: packages/components/carousel/src/carousel-item.vue [hunk 1]
```diff
@@ -110,9 +110,8 @@ function calcCardTranslate(index: number, activeIndex: number) {
     return (parentWidth * ((2 - CARD_SCALE) * (index - activeIndex) + 1)) / 4
   } else if (index < activeIndex) {
```
Comment (btea) [inline packages/components/carousel/src/carousel-item.vue:111]: The other statements here can also be removed

**Review by sxzz — COMMENTED** [review]
Hmmm I don't think we should change this code styles.

**Comment by sxzz** [comment]
IMHO, I think there are benefits to keeping the else, it's clearer.

**Pull request closed without merging**